wordle/src/main/kotlin/wordle/domain/Words.kt

```diff
@@ -0,0 +1,10 @@
+package wordle.domain
+
+class Words(
+    val value: List<Word>,
+) {
+
+    fun validateNonExist(word: Word) {
+        require(value.contains(word)) { "[예외] 사전에 존재하지 않는 단어입니다." }
+    }
```

Comment:
```suggestion
    fun validateNonExist(word: Word) {
        require(value.contains(word)) { "[예외] 사전에 존재하지 않는 단어입니다. (입력: ${word})" }
    }
```
코틀린엔 템플릿 문자열도 추가되어 가독성도 해치지 않는데, 위와 같이 디버깅을 위한 입력 데이터를 함께 출력해주는건 어떨까요?

Reply:
출력 뿐만 아니라 다방면으로 활용할 수 있겠네요 👍